Lib/test/test_email/test_email.py

```diff
@@ -1591,6 +1591,13 @@ def test_guess_minor_type(self):
             self._make_image(ext)
             subtype = ext if subtype is None else subtype
             self.assertEqual(self._im.get_content_type(), f'image/{subtype}')
+        
+    def test_guess_minor_type_other_jpeg_pattern(self):
+        # We can't have two files of the same name ending in
+        # '.jpg', so test the second valid pattern by passing
+        # the bytes directly.
+        _im = MIMEImage(b'\xff\xd8\xff\xdb')
+        self.assertEqual(_im.get_content_subtype(), 'jpeg')
```

Comment:
This will fix the CI error, which is complaining about trailing whitespace:

```suggestion

    def test_guess_minor_type_other_jpeg_pattern(self):
        # We can't have two files of the same name ending in
        # '.jpg', so test the second valid pattern by passing
        # the bytes directly.
        _im = MIMEImage(b'\xff\xd8\xff\xdb')
        self.assertEqual(_im.get_content_subtype(), 'jpeg')
```